arangod/ClusterEngine/ClusterIndex.cpp

```diff
@@ -123,6 +130,57 @@ void ClusterIndex::toVelocyPack(
     builder.add(StaticStrings::IndexEstimates, VPackValue(false));
   }
 
+  if (Index::hasFlag(flags, Index::Serialize::Figures)) {
```

Comment:
I don't understand. Is it possible to call "collection.getIndexes" and receive some index which is not finished building?

Reply:
Yes, to see their progress. But of course not so they are used for queries.